examples/simple_duck.py

```diff
@@ -0,0 +1,34 @@
+import sys
+
+import sycamore
+from sycamore.functions.tokenizer import HuggingFaceTokenizer
+from sycamore.llms import OpenAIModels, OpenAI
+from sycamore.transforms import COALESCE_WHITESPACE
+from sycamore.transforms.merge_elements import MarkedMerger
+from sycamore.transforms.partition import UnstructuredPdfPartitioner
+from sycamore.transforms.embed import SentenceTransformerEmbedder
+from sycamore.tests.config import TEST_DIR
+
+sys.path.append("../sycamore")
+
+model_name = "sentence-transformers/all-MiniLM-L6-v2"
+paths = str(TEST_DIR / "resources/data/pdfs/")
+
+OpenAI(OpenAIModels.GPT_3_5_TURBO_INSTRUCT.value)
+tokenizer = HuggingFaceTokenizer(model_name)
+
+ctx = sycamore.init()
+
+ds = (
+    ctx.read.binary(paths, binary_format="pdf")
+    .partition(partitioner=UnstructuredPdfPartitioner())
+    .regex_replace(COALESCE_WHITESPACE)
+    .mark_bbox_preset(tokenizer=tokenizer)
+    .merge(merger=MarkedMerger())
+    .spread_properties(["path"])
+    .split_elements(tokenizer=tokenizer, max_tokens=512)
+    .explode()
+    .embed(embedder=SentenceTransformerEmbedder(model_name=model_name, batch_size=100))
+    .sketch(window=17)
+)
+ds.write.duck()
```

Comment:
If we don't specify a location the data would go away after that process is killed, but even then we could print simple outputs that read from it before that? Otherwise we'd need to specify a location to persist it, from which we could directly open the database and show customers (don't necessarily need to write another reader for it)